tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    flake8
+    ; flake8 comment out for now, i'm unsure why it's needed though.
```

Comment:
I can't see the purpose of having flake8 here and it fails when used in CI.

Running tox locally runs OK through to the end.